gratipay/utils/fake_data.py

```diff
@@ -309,9 +309,14 @@ def populate_db(db, num_participants=100, ntips=200, num_teams=5, num_transfers=
 
     print("Making Teams")
     teams = []
+    num_teams -= 1      # Make room for the fake Grtipay Team 
```

Comment:
I don't think this is required, it should be okay if we have `num_teams` *inclusive* of the Gratipay team :)